packages/cli/src/commands/setup/chakra-ui/chakra-ui.js

```diff
@@ -9,7 +9,7 @@ import {
 } from './tasks/configure-storybook'
 import { checkSetupStatus, wrapWithChakraProvider } from './tasks/setup-chakra'
 
-export const command = 'chakra-ui'
+export const command = 'design chakra-ui'
```

Comment:
@thedavidprice I am not quite sure if this is the way you intended to nest this setup command. Maybe there is a better way with Listr?

Reply:
I would say for this PR, @TimKolberger maybe best to leave it at the top level, and we can move over the commands to be nested!

Reply:
Done ✅